pynicotine/pynicotine.py

```diff
@@ -140,12 +104,7 @@ def __init__(self, bindip, port):
         self.privileges_left = None
         self.manual_disconnect = False
 
-        self.server_conn = None
-        self.server_address = None
-        self.server_timer = None
-        self.server_timeout_value = -1
-
-        self.parent_conn = None
+        self.parent_socket = None
```

Comment:
consider `parent_link` instead of `parent_socket` , because socket is ambiguous with filesystem socks.